assets/dev/js/editor/regions/panel/pages/elements/views/search.js

```diff
@@ -13,12 +13,17 @@ PanelElementsSearchView = Marionette.ItemView.extend( {
 
 	events: {
 		'keyup @ui.input': 'onInputChanged',
+		'input @ui.input': 'triggerInputChange',
```

Comment:
I think this line is enough.
```suggestion
		'input @ui.input': 'onInputChanged',
```

Reply:
Well, you're right... It didn't work for me before for some reason, probably had some caching issue or something...